### PR TITLE
Change Country to an extended entity

### PR DESCRIPTION
**Pull request by SkaiBoa**
I send the pull request in answer of this discussion on the ORO-forums:
http://www.orocrm.com/forums/topic/extend-country-entity

Please review.

### CODE REVIEW

File: src/Oro/Bundle/AddressBundle/Model/ExtendCountry.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Oro\Bundle\AddressBundle\Model;
+
+class ExtendCountry 
+{
+public function __construct()
```
Comment (akuzmenko-magecore) [inline src/Oro/Bundle/AddressBundle/Model/ExtendCountry.php]: Hi. Thank you for your pull request.
Please, add spaces to indent the method declaration.
Also, we have a convention to add a comment that the real implementation of this method is autogenerated,
Please, refer to [ExtendAddress.php](https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/AddressBundle/Model/ExtendAddress.php ) for an example.

**Comment by SkaiBoa** [comment]
Sorry for my fault. I added the comment and the indentation.

**Comment by akuzmenko-magecore** [comment]
@SkaiBoa, we've merged your pull request into our development branch. The fix will be available in the upcoming release. Thank you.

**Comment by DimaSoroka** [comment]
Closing as address country is extended entity already

**Pull request closed without merging**